table/sst_file_writer.cc

```diff
@@ -231,6 +230,8 @@ Status SstFileWriter::Open(const std::string& file_path) {
   }
 
   sst_file->SetIOPriority(r->io_priority);
+  r->kv_validator = std::unique_ptr<OutputValidator>(
+      new OutputValidator(r->internal_comparator, true, true));
```

Comment:
Based on Google C++ Style we are following, we should comment on variable name for those two `true`s: https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments

Should we enable hash check only when r->mutable_cf_options.paranoid_file_checks is true?